package.json

```diff
@@ -72,10 +72,10 @@
     "@redhat-cloud-services/frontend-components-advisor-components": "^1.0.6",
     "@redhat-cloud-services/frontend-components-charts": "3.2.1",
     "@redhat-cloud-services/frontend-components-notifications": "^3.2.5",
-    "@redhat-cloud-services/frontend-components-pdf-generator": "2.6.7",
+    "@redhat-cloud-services/frontend-components-pdf-generator": "^2.6.7",
```

Comment:
If you update this to `^2.6.19`, you won't get those errors anymore.

Reply:
Im going to add this in there!